internal/os/os_test.go

```diff
@@ -0,0 +1,82 @@
+package os
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_OperatingSystem(t *testing.T) {
```

Comment:
End-to-end tests need to be [skipped in short mode](https://pkg.go.dev/testing#hdr-Skipping):

```suggestion
func TestCli_OperatingSystem(t *testing.T) {
    if testing.Short() {
        t.Skip("skipping test in short mode.")
    }
```